assets/js/app.js

```diff
@@ -40,6 +40,17 @@ topbar.config({barColors: {0: "#29d"}, shadowColor: "rgba(0, 0, 0, .3)"})
 window.addEventListener("phx:page-loading-start", _info => topbar.show(300))
 window.addEventListener("phx:page-loading-stop", _info => topbar.hide())
 
+window.addEventListener("dora:clipcopy", (event) => {
+  console.log(event)
```

Comment:
Do we need this console.log here?

Reply:
Nope, I forgot to remove it when debugging what `event` was. Nice catch